modules/virt-measuring-latency-vm-secondary-network.adoc

```diff
@@ -0,0 +1,252 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc
+
+:_content-type: PROCEDURE
+[id="virt-measuring-latency-vm-secondary-network_{context}"]
+= Checking network connectivity and latency for virtual machines on a secondary network
+
+As a cluster administrator, you use a predefined checkup to verify network connectivity and measure latency between virtual machines (VMs) that are attached to a secondary network interface. Set up the namespace and service account for the framework; create the `ClusterRole` and `ClusterRoleBinding` objects for the service account; enable permissions for the checkup; create the input config map; and run the checkup. After you record the results of the checkup, delete all the framework resources that you previously created.
```

Comment:
I think this first sentence would be a good module intro on its own, and the other details might be better in the "about" module (taking care to not make it look like a procedure).

One other note:

> After you record the results of the checkup, delete all the framework resources that you previously created.

The note at the bottom of the procedure ("Do not delete the framework and checkup role manifest files if you plan to run another checkup") makes me think that this should be rephrased to include the possibility that someone might run the checkup multiple times. Also, as it's currently written, this doc seems to imply that most people will only want to run the checkup once. Do you know if that reflects what most customers will do?

Reply:
@EdDev Can you please respond to Pan's question above?

Reply:
@EdDev I have updated the content to include the possibility that a user might run the checkup multiple times. Can you please review: http://file.rdu.redhat.com/sjhala/cnv-17454/virt/logging_events_monitoring/virt-running-cluster-checkups.html

cc: @ousleyp

Reply:
Well, it can be run multiple times as long as resources mentioned are removed.
So I guess this is correct.